hadoop-hdds/docs/content/start/StartFromDockerHub.zh.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Simple Single Ozone
+title: 简易 Ozone
```

Comment:
how about 'Single'? And from the context, what does Ozone mean here, like ozone server or ozone cluster?

Reply:
1. 'Single' is kind of implied by 'Simple'
2. docker-compose way is also introduced in this page, which is not so 'Single'
3. I cannot think of a good way to translate 'Single'

As for what Ozone means, I am not sure if we should emphasize that it is a cluster or server here, it kind of feels like "Ozone is Ozone".